ext/random/tests/02_engine/all_serialize_error.phpt

```diff
@@ -3,63 +3,65 @@ Random: Engine: serialize error pattern
 --FILE--
 <?php
```

Comment:
nits: I was wondering if I should add `declare(strict_types=1)`, although it makes no sense in this test case. What do you think?

Reply:
I don't think this is useful. In the interest of keeping the tests simple, everything that is not necessary should not be there. This allows the reader to focus on what's important.